app/src/components/parts/AuthenticateButton.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+    <div class="text-center">
+        <button type="submit" class="button is-info is-outlined">
+            {{buttonName}}
```

Comment:
`{`の両内側にはスペースをあけるのが一般的になっていますので、マスタッシュ構文のときにもそれにならいましょう。
他の部分も同様に修正しておきましょう。
```suggestion
            {{ buttonName }}
```